src/commands/org/delete/shape.ts

```diff
@@ -73,11 +73,13 @@ export class OrgShapeDeleteCommand extends SfCommand<OrgShapeDeleteResult | unde
       this.logSuccess(messages.getMessage('humanSuccess', [orgId]));
       this.log('');
       this.styledHeader('Failures');
```

Comment:
we can move this `styledHeader` into the `title:` property - check for all tables